src/main/proto/admin.proto

```diff
@@ -0,0 +1,169 @@
+syntax = "proto3";
+package io.axoniq.axonserver.grpc.admin;
+import "google/protobuf/empty.proto";
+option java_multiple_files = true;
+
+//************************ ContextAdminService *****************************
+
+service ContextAdminService {
+
+  rpc CreateContext(CreateContextRequest) returns (google.protobuf.Empty) ;
+
+  rpc DeleteContext(DeleteContextRequest) returns (google.protobuf.Empty) ;
+
+  rpc GetContext(ContextId) returns (ContextOverview) ;
+
+  rpc GetContexts(google.protobuf.Empty) returns (stream ContextOverview) ;
+
+  rpc SubscribeContextUpdates(google.protobuf.Empty) returns (stream ContextUpdate) ;
+
+}
+
+service ReplicationGroupAdminService {
+
+  rpc CreateReplicationGroup(ReplicationGroup) returns (google.protobuf.Empty);
```

Comment:
Change request type to CreateReplicationGroupRequest as the ReplicationGroup contains fields that are not used in the request and may lead to confusion (e.g. list of contexts)